### ISSUE TITLE
Prepare a survey (or GitHub Discussion about data sources)

### ISSUE THREAD

**Issue by zaleslaw** [Kotlin/dataframe#408]

**Comment by Jolanrensen** [comment]
Probably move this to a discussion so people can upvote and leave others :)

**Comment by Jolanrensen** [comment]
sure, but it might also be nice for the community to see which types of databases other people are interested in

**Comment by zaleslaw** [comment]
Nice to prepare the notebooks with the results:)

**Comment by zaleslaw** [comment]
@Jolanrensen will you share something?

**Comment by Jolanrensen** [comment]
Maybe we should add [Exposed](https://github.com/JetBrains/Exposed) to the list as a data source. It was suggested [here](https://github.com/Kotlin/dataframe/issues/28) first and seems to cover several DB types

**Comment by Jolanrensen** [comment]
Also, for people wanting to do heavy operations with lots of large columns, we might want to provide interop with [Multik](https://github.com/Kotlin/multik) as well

**Comment by koperagen** [comment]
Maybe something like Google Sheets

**Comment by Jolanrensen** [comment]
> Maybe something like Google Sheets

Like integration with their API? Could be easy, since we already have Excel support.

**Comment by koperagen** [comment]
> > Maybe something like Google Sheets
> 
> Like integration with their API? Could be easy, since we already have Excel support.

Yes, i think it might be a good step for building data processing pipelines. For example, read some data, transform with dataframe, write to a Google sheet. Or have a Google Sheet edited by a human and run dataframe processing on it when needed.

**Comment by Jolanrensen** [comment]
XML would also probably need OpenAPI support, similar to JSON

**Comment by belovrv** [comment]
I would also add yaml in the list
